dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/api/ContextAPIServlet.java

```diff
@@ -95,6 +95,7 @@ public void testContextStorage() {
         assertNotSame(Scope.noop(), scope);
         Context stored = storage.current();
         assertEquals(context, stored);
+        scope.close();
```

Comment:
This should be closed with a try block in case something throws an exception before we reach this line.